mp/src/game/client/CMakeLists.txt

```diff
@@ -0,0 +1,1693 @@
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_mxs.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_neobasecombatweapon.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_proxmine.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_proxmine.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_pz.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_pz.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_smac.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_smac.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_smokegrenade.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_smokegrenade.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_srm.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_srm.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_srms.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_srms.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_srs.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_srs.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_supa7.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_supa7.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_tachi.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_tachi.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_zr68c.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_zr68c.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_zr68l.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_zr68l.h
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_zr68s.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_zr68s.h
+)
+
+if(NEO_BUILD_WEAPON_PBK56S)
+    target_sources_grouped(
+        TARGET client
+        NAME "Source Files\\Shared\\Weapons"
+        FILES
+        ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_pbk56s.cpp
+        ${CMAKE_SOURCE_DIR}/game/shared/neo/weapons/weapon_pbk56s.h
+    )
+endif()
+
+target_sources_grouped(
+    TARGET client
+    NAME "HL2MP"
+    FILES
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/hl2mp_gamerules.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/hl2mp_gamerules.h
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/hl2mp_player_shared.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/hl2mp_player_shared.h
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/hl2mp_weapon_parse.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/hl2mp_weapon_parse.h
+    hl2mp/c_hl2mp_player.cpp
+    hl2mp/c_hl2mp_player.h
+    hl2mp/c_te_hl2mp_shotgun_shot.cpp
+    hl2mp/clientmode_hl2mpnormal.cpp
+    hl2mp/clientmode_hl2mpnormal.h
+)
+
+target_sources_grouped(
+    TARGET client
+    NAME "HL2MP\\Weapons"
+    FILES
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/weapon_357.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/weapon_ar2.cpp
+    ${CMAKE_SOURCE_DIR}/game/shared/hl2mp/weapon_ar2.h
```

Comment:
There's been some HL2MP weapons that been commented out in the vpc version recently by the HL2 weapons removal: https://github.com/NeotokyoRebuild/neo/commit/eeead46fa308754561fb64d24129b67be3c38b32#diff-967a8a275c86d528d08fa8c96557da8a2a3584dcfb9dd71c2adc72572964e42e 
Both client and server.

Reply:
Done